guides/v2.1/extension-dev-guide/routing.md

```diff
@@ -1,107 +1,185 @@
+## Router class
 
-<h3>Routers</h3>
+The [Router class] matches a request to an [action class] that processes the request.
 
-The routers information for the modules is described in the `routerList` parameter of <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/RouterList.php" target="_blank">Magento\Framework\App\RouterList</a> type in your `di.xml`.
+The following tables show the core routers that come with Magento:
 
-Each area has its own set of the routers. The `Magento\Framework\App\RouterList` model is injected into <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/FrontController.php" target="_blank">FrontController</a>.
+**`frontend` area routers:**
 
-You might need to customize the routers to change either the standard logic of processing the requests or the native Magento routers
-(such as, {% glossarytooltip f3944faf-127e-4097-9918-a2e9c647d44f %}CMS{% endglossarytooltip %} router, default router, and so on).
-However, you must not customize the routers that are used in Magento core modules.
+| Name         | Sort order | Description                                       |
+| ------------ | ---------- | ------------------------------------------------- |
+| [robots]     | 10         | Matches request to the `robots.txt` file          |
+| [urlrewrite] | 20         | Matches requests with URL defined in the database |
+| [standard]   | 30         | The standard router                               |
+| [cms]        | 60         | Matches requests for CMS pages                    |
+| [default]    | 100        | The default router                                |
+{:style="table-layout:auto"}
 
-<h3>Routes</h3>
+**`adminhtml` area routers:**
 
-Configurations of the routes are stored in `routes.xml` in the scopes area.
+| Name      | Sort order | Description                                |
+| --------- | ---------- | ------------------------------------------ |
+| [admin]   | 10         | Matches requests in the Magento admin area |
+| [default] | 100        | The default router for the admin area      |
+{:style="table-layout:auto"}
 
-Only the standard {% glossarytooltip b00459e5-a793-44dd-98d5-852ab33fc344 %}frontend{% endglossarytooltip %} and {% glossarytooltip 74d6d228-34bd-4475-a6f8-0c0f4d6d0d61 %}backend{% endglossarytooltip %} routers use routes. Typically, the configuration for a route is in the following format:
+### Standard router
 
-{% highlight XML %}
-<config>
-    <router id="%routerId%">
-        <route id="%routeId%" frontName="%frontName%">
-            <module name="%moduleName%" before="%moduleName%"/>
-        </route>
-    </router>
-</config>
-{% endhighlight %}
+A Magento {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} that uses the standard router has the following format: 
+
+```
+<store-url>/<front-name>/<controller-name>/<action-name>
```

Comment:
I think we should also include the optional store code in between `<store-url>` and `<front-name>` (which determines store/website context)